src/roller.ts

```diff
@@ -58,9 +58,19 @@ export abstract class Roller<D, F, P> implements IRoller {
     }
 
     public rollCommand(command: string): string {
+        const parts = command.split('#');
```

Comment:
Can you explain the need for the command splitting?

Reply:
That's a workaround for the FlavorText (since we didn't see that in the Documentation). We're going to change it to using the FlavorText.

Reply:
My intention here was to mimic the behaviour of the /roll command, which allows the following syntax: /roll {formula} # {comment}
Therefore splitting at '#'. 
shift() then extracts the command.
As the comment could contain more #, I joined the remaining parts together.
I'm sure this could be done more elegant with RegExp, but I was quicker this way.

The comment could then be simply passed as flavourText, which I wasn't aware of. After having a closer look, what still confuses me is the fact, that flavourText isn't used in any of the implementations other than passing it to the template. Did I miss some code or was your intention for it to be implemented in the XYRoller classes?

Reply:
It's used in the SW system for instance, going directly through API rather than parsing a chat message. 

So using a chat message directly doesn't work with flavorText

Reply:
This change will make flavourText also available in chat commands.